LABELS.md

```diff
@@ -0,0 +1,112 @@
+# Labels
+
+The following lays out common and recommended labels to use with
+System Containers.
+
+
+## Architecture
+
+Describes the architecture of the image.
+
+This label is RECOMMENDED.
+
+### Example
+```
+Architecture="x86_64"
+```
+
+
+## atomic.type
+
+Denotes the type of image to the atomic command. If the image is for
+system containers then this label should be provided and set to system.
+
+This label MUST be used if the image is for use as a system container.
```

Comment:
Perhaps we should clarify what this label entails (pull to ostree, install as system container) and note that it is for images that are ONLY for system containers, since images such as etcd can be used as both docker and system.